index.html

```diff
@@ -15,16 +15,44 @@
   </head>
   <body>
     <main class="main">
+      <nav>
```

Comment:
- [ ] I have noticed that the mobile menu does not take the full width, please try to match it with Figma design: 

Figma | Yours
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/147062956-e1f72ac4-f541-4481-92db-14136bf76561.png) | ![image](https://user-images.githubusercontent.com/77038610/147061496-ba69cf2d-fc4d-4a40-a88d-9aeb25c206b6.png)

- [ ] I can also not see the `x` button which is supposed to close the menu. Please add the `x` button, this is happening because of:

```
.toggle-menu {
 left: 145px
}
```

If you delete ` left: 145px` both issues will be solved 😉